### PR TITLE
Feature/profile page changes

### PR DESCRIPTION
**Pull request by Bluesmile82**
Changes to the profile page based on new desings. The designs for regions are still missing.
https://projects.invisionapp.com/share/Y5SOCAVT7GD#/screens/370547066

image
image
image

### CODE REVIEW

File: frontend/scripts/react-components/profile-node/profile-node-widgets/links-widget.component.jsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+function LinksWidget(props) {
+  const { year, nodeId, contextId, profileType } = props;
+  const params = { node_id: nodeId, context_id: contextId, profile_type: profileType, year };
+  return (
+    <Widget params={[params]} query={[GET_NODE_SUMMARY_URL]}>
+      {({ data, loading, error }) => {
+        if (loading) {
+          return (
+            <div className="section-placeholder" data-test="loading-section">
```
Comment (sorodrigo) [inline frontend/scripts/react-components/profile-node/profile-node-widgets/links-widget.component.jsx]: I wouldnt add a spinner for this

File: frontend/scripts/react-components/profiles/button-links/button-links.component.jsx [hunk 2]
```diff
@@ -1,92 +1,88 @@
...
-                serializerParams: {
+    <div className="c-button-links">
+      <div className="row button-links-row">
+        <Link
+          className="link-button"
+          to={{
+            type: 'tool',
+            payload: {
+              query: {
+                state: {
```
Comment (sorodrigo) [inline frontend/scripts/react-components/profiles/button-links/button-links.component.jsx]: please rebase from develop, this is no longer valid. 🙊

File: frontend/scripts/react-components/profiles/button-links/button-links.scss [hunk 3]
```diff
@@ -2,68 +2,67 @@
...
+    justify-content: space-between;
     width: 100%;
     height: 100%;
     min-height: 70px;
+    padding: 5px 20px;
+    flex: 1;
+    margin-bottom: 20px;
+    margin-left: 0;
+
+    @media screen and (min-width: $breakpoint-foundation-medium) {
```
Comment (sorodrigo) [inline frontend/scripts/react-components/profiles/button-links/button-links.scss]: why not use the grid?

File: frontend/scripts/tests/puppeteer/shared.js [hunk 4]
```diff
@@ -1,34 +1,53 @@
...
+  if (title) {
+    waitPromises.push(page.waitForSelector('[data-test=profiles-title]'));
+    contentPromises.push(page.$eval('[data-test=profiles-title]', t => t.textContent));
+  }
+
+  await Promise.all(waitPromises);
+  const promiseResult = await Promise.all(contentPromises);
+
+  if (title) {
+    expect(last(promiseResult)).toBe(title);
```
Comment (sorodrigo) [inline frontend/scripts/tests/puppeteer/shared.js]: you keep surprising me with new cool lodash methods 😛

File: frontend/scripts/react-components/profile-node/profile-node-widgets/links-widget.component.jsx [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+        }
+
+        const name =
+          data.GET_NODE_SUMMARY_URL.nodeName || data.GET_NODE_SUMMARY_URL.jurisdictionName;
+
+        return (
+          <section className="c-links-widget columns">
+            <div className="row">
+              <Heading as="h3" weight="bold" variant="mono" size="md">
+                Explore other information relevant to {name}
```
Comment (sorodrigo) [inline frontend/scripts/react-components/profile-node/profile-node-widgets/links-widget.component.jsx]: don't forget the use of `translateText`

**Comment by Bluesmile82** [comment]
Ready again! Definitely, I should thoroughly review those snapshots. Thanks for pointing it out!

**Comment by sorodrigo** [comment]
The heading misses a padding before the section block and also the heading should be hidden in print mode
Screen Shot 2019-07-04 at 19 13 57
Screen Shot 2019-07-04 at 19 13 01

**Comment by Bluesmile82** [comment]
Fixed finally

**Pull request merged**